e2e/common/types.ts

```diff
@@ -65,7 +65,10 @@ export type fl401SolicitorEvents =
   | "View PDF application"
   | "Statement of truth and submit";
 
-export type c100CaseWorkerActions = "Manage orders";
+export type c100CaseWorkerActions =
```

Comment:
these are fl401 case progression items (although i'm pretty sure they will be similar to c100 aswell - might be worth combining with other types?